src/igvfd/types/lab.py

```diff
@@ -22,3 +23,14 @@ class Lab(Item):
     schema = load_schema('igvfd:schemas/lab.json')
     name_key = 'name'
     embedded = ['awards']
+
+    @calculated_property(
+            schema={
+                'title': 'Title',
+                'type': 'string',
+            }
+        )
+    def title(self, request, pi, institute_label):
+        pi_object = request.embed(pi, '@@object')
+        pi_name = pi_object.get('title')
+        return(pi_name + ', ' + institute_label)
```

Comment:
Think should prefer f-strings where possible: 

`return f'{pi_name}, {institute_label}'`